### ISSUE TITLE
Collection.store eventually fails after adding records inside with db.transaction()

### ISSUE THREAD

**Issue by kolbyjack** [coleifer/unqlite-python#22]
I'm trying to convert a small project from an sqlite to unqlite database, and ran into an issue while trying to import the data from sqlite.  After importing 409 events into a collection, Collection.store reliably fails with the error "ValueError: Error fetching return value from script"

Here's a minimal script that can reproduce the issue:

```python
#!/usr/bin/env python

import unqlite

unq = unqlite.UnQLite('test.unq')

events = unq.collection('events')
events.create()
for i in xrange(10000):
    with unq.transaction():
        print events.fetch(events.store({'foo': 'bar'}))
```

it dies for me after adding 1016 events:

```
...
{'foo': 'bar', '__id': 1014}
{'foo': 'bar', '__id': 1015}
Traceback (most recent call last):
  File "./test.py", line 11, in <module>
    print events.fetch(events.store({'foo': 'bar'}))
  File "unqlite.pyx", line 1141, in unqlite.Collection.store (unqlite.c:18202)
  File "unqlite.pyx", line 1046, in unqlite.Collection._simple_execute (unqlite.c:16851)
  File "unqlite.pyx", line 1054, in unqlite.Collection._simple_execute (unqlite.c:16789)
ValueError: Error fetching return value from script.
```

**Comment by kolbyjack** [comment]
Nevermind, this was fixed by the latest commit, installing directly from the repo fixed the issue

**Comment by coleifer** [comment]
Released 0.5.2 if you prefer to point at that instead.
